src/components/ManageUsers/UserItem.jsx

```diff
@@ -0,0 +1,23 @@
+function UserItem({ user }) {
+  const getIcon = () => {
+    if (user.role === 'admin') {
+      return 'admin.png';
+    } else if (user.role === 'librarian') {
+      return 'librarian.png';
+    } else {
+      return 'customer.png';
+    }
+  };
+  const icon = getIcon();
+  return (
+    <li className="useritem">
```

Comment:
2-word classes should be separated by `-`. so in this case, it should be
```suggestion
    <li className="user-item">
```
Please change it in all places

Reply:
name class "useritem" was changed to "user-item" in each recurence.